apps/condo/domains/organization/hooks/useCreateOrganizationModalForm.tsx

```diff
@@ -138,6 +139,10 @@ export const useCreateOrganizationModalForm = ({ onFinish }: IUseCreateOrganizat
         setIsVisible(false)
     }, [setIsVisible])
 
+    const inputStyle = {
+        backgroundColor: '#F0F0F0',
+    }
```

Comment:
Please, extract this style to `apps/condo/domains/common/constants/style.js` and name it as `inputOnPopupStyle`.
I will discuss with design-team the reason of this fill color, it looks not consistent with other forms…

Reply:
done